maint-lib/stagelib/envglobals.py

```diff
@@ -8,25 +8,25 @@
 
 import stagelib.git as git
 
-ALIGNED_NEWLINE = '\n' + ' '*21  # align second line to column
+ALIGNED_NEWLINE = '\n' + ' ' * 21  # align second line to column
 # Ordered dict with format
 # <var name>: <description>
 # Add new required variables simply by adding a new tuple to this setup
-# Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols
+# Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols  # noqa E501
 REQUIRED_ENV_VARS = OrderedDict([
-    ('CEPH_VERSION',       'Ceph named version being built (e.g., luminous, mimic)'),
-    ('CEPH_POINT_RELEASE', 'Points to specific version of ceph (e.g -12.2.0) or empty'),
-    ('HOST_ARCH',          'Architecture of binaries being built (e.g., amd64, arm32, arm64)'),
-    ('BASEOS_REG',         'Registry for the container base image (e.g., _ (x86_64), arm64v8 (aarch64))' +  # noqa: E501
-                           ALIGNED_NEWLINE + 'There is a relation between HOST_ARCH and this value'),  # noqa: E501
-    ('BASEOS_REPO',        'Repository for the container base image (e.g., ubuntu, opensuse)'),  # noqa: E501
-    ('BASEOS_TAG',         'Tagged version of BASEOS_REPO container (e.g., 16.04, 42.3 respectively)'),  # noqa: E501
-    ('IMAGES_TO_BUILD',    'Container images to be built (usually should be "dockerfile daemon")'),
-    ('STAGING_DIR',        'Dir into which files will be staged' + ALIGNED_NEWLINE +
-                           'This dir will be overwritten if it already exists'),
-    ('RELEASE',            'Release string for the build'),
-    ('DAEMON_BASE_IMAGE',  'Tag given to the daemon-base image and used as base for the daemon image'),  # noqa: E501
-    ('DAEMON_IMAGE',       'Tag given to the daemon image'),
+    ('CEPH_VERSION',       'Ceph named version being built (e.g., luminous, mimic)'),  # noqa: E241,E501
```

Comment:
Ignoring E241 should be unnecessary here. I think this may be a product of running an old version of flake8